versions.props

```diff
@@ -58,7 +58,7 @@
     <HealthChecksRedisVersion>2.2.3</HealthChecksRedisVersion>
     <HealthChecksSqlServerVersion>2.2.0</HealthChecksSqlServerVersion>
 
-    <SonarAnalyzerVersion>8.29.0.36737</SonarAnalyzerVersion>
+    <SonarAnalyzerVersion>8.32.0.39516</SonarAnalyzerVersion>
```

Comment:
In case it helps: I ran my ruleset scanner on both SonarAnalyzer DLL versions. It produces the next diff:
```
+ S3059: None
+ S3260: Warning
+ S3267: Warning
```
Makes sense to suppress the new rules in 3.x, though we should be cautious when merging 3.x into main and ensure we don't suppress them in v4.x.

Reply:
I think the things that were flagged were all fixed in [this commit](https://github.com/SteeltoeOSS/Steeltoe/pull/891/commits/2f26d5710c910fa57ca11140bbf2ecde38cb1a42). Admittedly this commit got a little murky as it seems like the file-scoped namespace changes didn't adjust the whitespace inside conditional comments, so I just fixed it all at once since it seemed quite minor

Reply:
> the file-scoped namespace changes didn't adjust the whitespace inside conditional comments

Yes, I ran into that too.

Do you expect merges from 3.x to main, aside from cherry-picking? If so, perhaps add a comment to not propagate those changes into main?

Reply:
I do not expect any merges from 3.x to main aside from cherry-picking